sdk/src/main/java/io/dapr/client/AbstractClientBuilder.java

```diff
@@ -5,55 +5,11 @@
 package io.dapr.client;
 
 import io.dapr.utils.Constants;
+import okhttp3.OkHttpClient;
 
 /**
  * Base class for client builders
  */
 public abstract class AbstractClientBuilder {
```

Comment:
Delete.

Reply:
Forgot to delete this class, which is no longer required.